source/installation-guide/installation-guide.rst

```diff
@@ -259,6 +259,8 @@ OS2IoT-backend takes several environment variables as configuration, if these ar
 +-------------------------------+------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------+
 | CA_KEY_PASSWORD               | The password for the Certificate Authority key.                                                      | :code:`os2iot`                                                                          |
 +-------------------------------+------------------------------------------------------------------------------------------------------+-----------------------------------------------------------------------------------------+
+| CHIRPSTACK_API_KEY            | The API key (Bearer token) created in Chirpstack.                                                    | :code:`apikey`                                                                          |
```

Comment:
Is the `CHIRPSTACK_JWTSECRET` described further up still relevant?

Reply:
Actually, no. I think that could be removed.